drivers/clock_control/clock_control_mcux_ccm_rev2.c

```diff
@@ -114,6 +114,23 @@ static int mcux_ccm_get_subsys_rate(const struct device *dev,
 	return 0;
 }
 
+static int mcux_ccm_init(const struct device *dev)
+{
+#ifdef CONFIG_SOC_MIMX93_A55
```

Comment:
I'm unclear why this change is needed- is this variable declared in the HAL?

Reply:
These clock source frequency need to be initialized in hal driver, we already have patch in mcux-sdk's internal development repo, so push it to github repo and port the fix to hal_nxp firstly before we merge this PR.

Reply:
I've port it to hal_nxp. see pr: 
https://github.com/zephyrproject-rtos/hal_nxp/pull/286